__tests__/apiTopics.test.js

```diff
@@ -0,0 +1,27 @@
+const request = require("supertest");
+const app = require("../index");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+afterAll(() => {
+  db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+describe("Get / api/topics", () => {
+  it("returns with array of objects in topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .then((response) => {
+        expect(response.status).toBe(200);
+        expect(response.body).toBeInstanceOf(Array);
+        response.body.forEach((topic) => {
+          expect(topic).toHaveProperty("slug");
+          expect(topic).toHaveProperty("description");
+        });
+      });
+  });
+});
```

Comment:
Great test, can you add a sad path test for if the user spells the path wrong, for example `/api/toopics` 
This should give a 404 and an error message